exampleapp/src/test/java/com/stormpath/sdk/exampleapp/ExampleUnitTest.java

```diff
@@ -0,0 +1,16 @@
+package com.stormpath.sdk.exampleapp;
```

Comment:
don't think we need this file. If we want to provide a unit test in the example app, it should be testing something in the exampleaap.